react/src/lib/components/DeckGLMap/layers/utils/propertyMapTools.ts

```diff
@@ -1,7 +1,7 @@
-import { PickInfo } from "@deck.gl/core/lib/deck";
 import { RGBColor } from "@deck.gl/core/utils/color";
+import { LayerPickInfo } from "../../components/InfoCard";
```

Comment:
In the future, the map component could be packaged by itself, so we shouldn't have dependencies on other internal components unless they are really used in the map.

You can move the needed classes (LayerPickInfo and PropertyDataType) into a file in layers/utils.